### DISCUSSION TITLE
Comment on the status of Nothing in the move from Becoming to Determinate Being

### DISCUSSION THREAD

**Discussion by ridetoruin** [systemphil/sphil#81] [Hegel]
This comment is in response to the interpretation put forward by @Firgrep in the article on "Becoming", under the sub-heading of "Not Nothing", and concerns the status of the determination of nothing in this moment of the logical development.

The idea that the unity falls under header of "being" because this unity 'is' raises some questions. First of all, doesn't "nothing" also 'is' insofar as it is a determination of thought that is being investigated? It must have being (and, in fact, I think that's Hegel's point about the concept of Nothing), and so it must have being, in other words, it must 'is'. It is in this sense, that I think Hegel is using "being" as a shorthand for both "being" and "nothing". In fact, he's using it as a shorthand for determinate being. Just as when he distinguishes it as a "determination on its own" and a "determination of the whole".

If the above is wrong, however, what does that tell us about the the determination of "nothing"? Why is it such a central determination in our understanding of "becoming" and the moments of coming-to-be and ceasing-to-be, but is irrelevant in the moment that follows from them? I'm not sure that quite makes sense.

All of the above, however, is nullified by what Hegel says in the next paragraph:

"Becoming, as this transition into the unity of being and nothing, a unity which is in the form of being or has the form of the onesided immediate unity of these moments, is determinate being." (106, Miller).

Hegel assuages all of the concerns I have raised above by saying that the form that it has is a onesided one - i.e it ignores nothing. That is, at least, consistent. But, at this stage, I suppose my concern would be with the status of "nothing". What does Hegel mean by the form of being? Does he mean is-ness? Well, surely "nothing" has isness, otherwise, how do we think it? What exactly is meant by the form of being? Which by the way, sounds like a very determinate understanding of such an indeterminate determination.

Also, now that I am re-reading this section, I would query the following sentence: " it is being that comes-to-be and ceases-to-be". That is not quite right, it is nothing that comes to be and being that ceases to be.

**Comment by Firgrep** [comment]
For reference, here is [the article in question](https://sphil.xyz/hegel/reference/becoming/development) 

EDIT: now public

**Comment by Firgrep** [comment]
Thinking about this further. 

> First of all, doesn't "nothing" also 'is' insofar as it is a determination of thought that is being investigated? It must have being (and, in fact, I think that's Hegel's point about the concept of Nothing), and so it must have being, in other words, it must 'is'. It is in this sense, that I think Hegel is using "being" as a shorthand for both "being" and "nothing". In fact, he's using it as a shorthand for determinate being. Just as when he distinguishes it as a "determination on its own" and a "determination of the whole".

1. I'm not sure you can determine that "nothing is" becuase that exactly vanishes the nothing&mdash;it is no longer nothing the moment it is! And if we persist in saying that `nothing` must have being, we are already beyond the point of `pure nothing` and have already thought of it as inseparable from `being`, and so it is something existant. 

> If the above is wrong, however, what does that tell us about the the determination of "nothing"? Why is it such a central determination in our understanding of "becoming" and the moments of coming-to-be and ceasing-to-be, but is irrelevant in the moment that follows from them? I'm not sure that quite makes sense.

2. Why do you think it becomes irrelevant? It is `nothing`! It is essential. Rather, it becomes transformed when taken up in a new context (to non-being). 

> "Becoming, as this transition into the unity of being and nothing, a unity which is in the form of being or has the form of the onesided immediate unity of these moments, is determinate being." (106, Miller).
Hegel assuages all of the concerns I have raised above by saying that the form that it has is a onesided one - i.e it ignores nothing. That is, at least, consistent. But, at this stage, I suppose my concern would be with the status of "nothing". What does Hegel mean by the form of being? Does he mean is-ness? Well, surely "nothing" has isness, otherwise, how do we think it? What exactly is meant by the form of being? Which by the way, sounds like a very determinate understanding of such an indeterminate determination.

3. Insofar as we say that the unity _is_, being takes precedence yes.
4. The "status" of nothing is that it must be implied and that's what's revealed in the non-being bit, what becomes negation. 
5. "Form of being", I suppose being is understood as the main context here where being and nothing are its moments?
6. "Nothing has isness, otherwise, how do we think it?" See my point 1. But as to "how do we think it", that's going off topic here, isn't it? Does my point 1 help any in that regard?

> Also, now that I am re-reading this section, I would query the following sentence: " it is being that comes-to-be and ceases-to-be". That is not quite right, it is nothing that comes to be and being that ceases to be.

7. This is where I disagree. How about the following formulation? It is `being` that comes **to be** and caeses **to be** whereas `nothing` **comes** to be and **ceases** to be. Here `nothing` is _in_ what is not there (yet), where no `being` is, where negativity reigns.
